lib/init.g

```diff
@@ -937,6 +937,9 @@ InstallAndCallPostRestore( function()
         f := GAPInfo.InitFiles[i];
         if IsRecord(f) then
             status := READ_NORECOVERY(InputTextString(f.command));
+        elif EndsWith(f, ".tst") then
+            Test(f);
+            status := true;
```

Comment:
Of course one could try to make this more robust: e.g. first check if `f` refers to an existing and readable file, and if not... dunno, what? Right now, one gets a break loop, which says "type 'quit;' to quit to outer loop" but if one does that, GAP actually quits (because the main REPL has not yet been started). Hmm